File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        k = nums.copy()
        k.sort()
        if(k == nums):
            return True
        n = False
        for i in range(0,len(nums)):
            a = k[-1]
            k.pop()
            k.insert(0,a)
            if(k == nums):
                return True
        return n

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        vector<int> k = nums;
        sort(k.begin(),k.end());
        if(nums == k){
            return true;
        }
        bool w = false;
        for(int i =0;i<nums.size();i++){
            int a = k.back();
            k.pop_back();
            k.insert(k.begin(),a);
            if(nums == k){
                w = true;
            }

        }
        return w;

    }
};